TestNamespacesListForbidden flaky on CI

On downstream CI I noticed:

```
  --- FAIL: TestNamespaces/TestNamespacesListForbidden (2.08s)
      --- FAIL: TestNamespaces/TestNamespacesListForbidden/namespaces_list_(forbidden) (2.01s)
          --- FAIL: .../returns_error (0.00s)
          --- FAIL: .../sends_log_notification (2.01s)
```

The itself test uses a 2-second timeout, sometimes this than fails. Idea is to enhance that timeout

--- a/pkg/mcp/namespaces_test.go
+++ b/pkg/mcp/namespaces_test.go
@@ -101,7 +101,7 @@ func (s *NamespacesSuite) TestNamespacesListForbidden() {
 				"error message should indicate forbidden")
 		})
 		s.Run("sends log notification", func() {
-			logNotification := capture.RequireLogNotification(s.T(), 2*time.Second)
+			logNotification := capture.RequireLogNotification(s.T(), 5*time.Second)
 			s.Equal("error", logNotification.Level, "forbidden errors should log at error level")
 			s.Contains(logNotification.Data, "Permission denied", "log message should indicate permission denied")
 		})

